datus/models/openai_compatible.py

```
@@ -842,9 +843,43 @@ async def _stream_operation():
+                        messages=f"Connection interrupted, retrying ({attempt + 2}/{max_retries})...",
+                        input_data={"error": str(e), "attempt": attempt + 1},
+                        status=ActionStatus.PROCESSING,
+                    )
+                    action_history_manager.add_action(retry_action)
+                    yield retry_action
+
+                    await asyncio.sleep(retry_delay)
+                    retry_delay *= 2  # Exponential backoff
+                else:
+                    # All retries exhausted
+                    logger.error(f"Stream failed after {max_retries} attempts: {type(e).__name__}: {e}")
```

Ruff (0.14.6)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)